packages/treasury/src/stablecoinMachine.js

```diff
@@ -92,13 +93,20 @@ export async function start(zcf) {
   // we assume the multipool-autoswap is public, so folks can buy/sell
   // through it without our involvement
   // Should it use creatorFacet, creatorInvitation, instance?
-  /** @type {{ publicFacet: MultipoolAutoswapPublicFacet, instance: Instance}} */
-  const { publicFacet: autoswapAPI, instance: autoswapInstance } = await E(
-    zoe,
-  ).startInstance(
+  /** @type {{ publicFacet: MultipoolAutoswapPublicFacet, instance: Instance,
+   *  creatorFacet: MultipoolAutoswapCreatorFacet }} */
+  const {
+    publicFacet: autoswapAPI,
+    instance: autoswapInstance,
+    creatorFacet: autoswapCreatorFacet,
+  } = await E(zoe).startInstance(
     autoswapInstall,
     { Central: runIssuer },
-    { timer: timerService },
+    {
+      timer: timerService,
+      poolFee: loanParams.poolFee,
```

Comment:
The treasury should not be so informed about the structure of the AMM. We will need to refactor the creation so they are instantiated more independently.

Reply:
added #3019